go.mod

```diff
@@ -40,6 +39,7 @@ require (
 	github.com/mattermost/logr/v2 v2.0.15
 	github.com/mattermost/morph v0.0.0-20220401091636-39f834798da8
 	github.com/mattermost/rsc v0.0.0-20160330161541-bbaefb05eaa0
+	github.com/mattermost/squirrel v0.2.0
```

Comment:
`v0.1.0` was the last "release", and `v0.2.0` renames to `mattermost/squirrel`. I have some other pending (and breaking changes), so keeping this < v1.